src/main/java/org/example/C03_11_PatternAngledTriangleRepeatedNumberInRow.java

```diff
@@ -0,0 +1,19 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class C03_11_PatternAngledTriangleRepeatedNumberInRow {
```

Comment:
"AngledTriangle" does not make much sense. Better "RightAngledTriangle"

Reply:
done

Reply:
Now is worse. Give a name that describes the problem.

Reply:
Now?